packages/amplify-data-construct/package.json

```diff
@@ -0,0 +1,167 @@
+{
+  "name": "@aws-amplify/data-construct",
+  "version": "1.3.0",
```

Comment:
Have you found a way to keep the versions in sync?

Reply:
Nothing explicit, since lerna manages our versions.